javascriptlib/src/mill/javascriptlib/TestModule.scala

```diff
@@ -0,0 +1,147 @@
+package mill.javascriptlib
+
+import mill.*
+import os.*
+
+trait TestModule extends TaskModule {
+  import TestModule.TestResult
+
+  def test(args: String*): Command[Seq[TestResult]] =
```

Comment:
All the implementations right now return `Nil`. If we don't have anything to return, let's just make it return `Command[Unit]` for now

Reply:
returning Command[TestResult]
`type TestResult = Unit`